product_packaging_level_salable/tests/test_packaging_level_can_be_sold.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo.exceptions import ValidationError
+
+from .common import Common
+
+
+class TestPackagingLevelCanBeSold(Common):
+    @classmethod
+    def setUpClassSaleOrder(cls):
+        super().setUpClassSaleOrder()
+        cls.order_line.product_uom_qty = 3.0
+        # Needed for W8110 of pylint-odoo.
+        return None
+
+    def test_packaging_level_can_be_sold(self):
+        self.order_line.write({"product_packaging_id": self.packaging_tu.id})
+        with self.assertRaises(ValidationError):
+            self.order_line.write(
+                {"product_packaging_id": self.packaging_cannot_be_sold.id}
+            )
+            onchange_res = self.order_line._onchange_product_packaging_id()
+            self.assertIn("warning", onchange_res)
```

Comment:
These lines are never tested because of the ValidationError just above.

Could you move this to a separate test method?